gitbook-docs/market/front-end.md

```diff
@@ -1,137 +1,137 @@
 ---
-description: Proof Market front end guide
+description: The web interface of the Proof Market
```

Comment:
I have a feeling that we can omit 'the' in headings.

Reply:
fixed